posts/2019-02-20-framework-release-v138-websockets.md

```diff
@@ -0,0 +1,161 @@
+- [#5740](https://github.com/serverless/serverless/pull/5740) Provide multi origin cors values<a href="https://github.com/serverless/serverless/pull/5740/files?utf8=✓&diff=split" style="text-decoration:none;"> <span style="color:#28a647">+85</span>/<span style="color:#cb2431">-11</span></a> <a href="https://github.com/richarddd"> <img src='https://avatars2.githubusercontent.com/u/1422927?v=4' style="vertical-align: middle" alt='' height="20px"> richarddd</a>
+- [#5754](https://github.com/serverless/serverless/pull/5754) Add Hello World Ruby Example<a href="https://github.com/serverless/serverless/pull/5754/files?utf8=✓&diff=split" style="text-decoration:none;"> <span style="color:#28a647">+63</span>/<span style="color:#cb2431">-0</span></a> <a href="https://github.com/yuki3738"> <img src='https://avatars3.githubusercontent.com/u/6305192?v=4' style="vertical-align: middle" alt='' height="20px"> yuki3738</a>
+- [#5758](https://github.com/serverless/serverless/pull/5758) AWS: Add fallback support in ${cf} and ${s3}<a href="https://github.com/serverless/serverless/pull/5758/files?utf8=✓&diff=split" style="text-decoration:none;"> <span style="color:#28a647">+137</span>/<span style="color:#cb2431">-1</span></a> <a href="https://github.com/exoego"> <img src='https://avatars2.githubusercontent.com/u/127635?v=4' style="vertical-align: middle" alt='' height="20px"> exoego</a>
+
+#### Documentation
+- [#5731](https://github.com/serverless/serverless/pull/5731) Fix link<a href="https://github.com/serverless/serverless/pull/5731/files?utf8=✓&diff=split" style="text-decoration:none;"> <span style="color:#28a647">+7</span>/<span style="color:#cb2431">-7</span></a> <a href="https://github.com/kazufumi-nishida-www"> <img src='https://avatars0.githubusercontent.com/u/40748597?v=4' style="vertical-align: middle" alt='' height="20px"> kazufumi-nishida-www</a>
+- [#5751](https://github.com/serverless/serverless/pull/5751) Fix typo in Multiple Configuration Files example<a href="https://github.com/serverless/serverless/pull/5751/files?utf8=✓&diff=split" style="text-decoration:none;"> <span style="color:#28a647">+1</span>/<span style="color:#cb2431">-1</span></a> <a href="https://github.com/paflopes"> <img src='https://avatars0.githubusercontent.com/u/5330156?v=4' style="vertical-align: middle" alt='' height="20px"> paflopes</a>
+- [#5788](https://github.com/serverless/serverless/pull/5788) Document how to use Secrets Manager<a href="https://github.com/serverless/serverless/pull/5788/files?utf8=✓&diff=split" style="text-decoration:none;"> <span style="color:#28a647">+18</span>/<span style="color:#cb2431">-0</span></a> <a href="https://github.com/dschep"> <img src='https://avatars0.githubusercontent.com/u/667763?v=4' style="vertical-align: middle" alt='' height="20px"> dschep</a>
+
+#### Roadmap and focus
+
+The next release plans to maintain our cadence and keep tackling our issue and PR backlog.
```

Comment:
Maybe link to the next milestone(s) here --> https://github.com/serverless/serverless/milestone/61 and https://github.com/serverless/serverless/milestone/62

Those will be filled during our upcoming call today.